src/palace/manager/service/celery/celery.py

```diff
@@ -44,204 +44,213 @@ def beat_schedule() -> dict[str, Any]:
     be a timedelta object or a crontab string. The `task` key is required and should be the name
     of the task to run.
     """
+    from palace.manager.celery.tasks import (
```

Comment:
We have to import the tasks within the function to avoid an import loop. I think it still makes sense to reference the tasks by their function, so its easier to notice when they are removed, or refactored, instead of just referring to them by string.